python/cli/prelude_cli/views/iam.py

```diff
@@ -102,117 +117,128 @@ def detach_oidc(controller):
-    help="override the profile account id",
-    default=None,
-    show_default=False,
-    type=str,
-)
 @click.pass_obj
 @pretty_print
-def reset_password(controller, email, account):
-    """Reset a user's password"""
-    with Spinner(description="Resetting password"):
-        controller.reset_password(email=email, account_id=account)
-
-    token = click.prompt(
-        "Check your email for the reset token.\nEnter your reset token", type=str
-    )
-    with Spinner(description="Retrieving new password"):
-        return controller.verify_user(token=token)
+def create_service_user(controller, email):
+    """Create a new service user in your account"""
+    with Spinner(description="Creating new service user"):
+        data = controller.create_service_user(email=email)
+    msg = "New user must check their email to verify their account"
+    return data, msg
 
 
 @iam.command("update-user")
-@click.option(
-    "-d", "--days", help="days from now this user will remain active", type=int
-)
+@click.option("-n", "--name", help="name of user", type=str)
+@click.pass_obj
+@pretty_print
+def update_user(controller, name):
+    """Update your user information"""
+    with Spinner(description="Updating user"):
+        return controller.update_user(name=name)
+
+
+@iam.command("update-account-user")
 @click.option(
     "-p",
     "--permission",
     help="user permission level",
+    default=Permission.EXECUTIVE.name,
     type=click.Choice(
         [
             p.name
             for p in Permission
-            if p not in [Permission.INVALID, Permission.SERVICE]
+            if p not in [Permission.INVALID, Permission.SERVICE, Permission.SUPPORT]
         ],
         case_sensitive=False,
     ),
+    show_default=True,
+)
+@click.option(
+    "-o", "--oidc", help="OIDC app name", default=None, show_default=False, type=str
 )
-@click.option("-n", "--name", help="name of user", type=str)
-@click.option("--oidc/--no-oidc", help="whether user must login via SSO", default=None)
 @click.argument("email")
 @click.pass_obj
 @pretty_print
-def update_user(controller, days, permission, name, oidc, email):
+def update_account_user(controller, permission, oidc, email):
     """Update a user in your account"""
-    with Spinner(description="Updating user"):
-        return controller.update_user(
-            email=email,
-            permission=Permission[permission] if permission else None,
-            name=name,
-            expires=datetime.now(timezone.utc) + timedelta(days=days) if days else None,
-            oidc=oidc,
+    with Spinner(description="Updating account user"):
+        return controller.update_account_user(
+            email=email, oidc=oidc, permission=permission
         )
 
 
-@iam.command("delete-user")
+@iam.command("remove-user")
 @click.confirmation_option(prompt="Are you sure?")
-@click.argument("handle")
+@click.option(
+    "-o", "--oidc", help="OIDC app name", default=None, show_default=False, type=str
```

Comment:
think we should make oidc a required flag everywhere so folks don't accidentally target the wrong user?